l10n_it_fatturapa_in/tests/data/IT05979361218_011.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Se faccio il test su SDI di questo file ottengo
```
Invalid IdFiscaleIVA of "CedentePrestatore" | Check that the Partita IVA (VAT Number) field of the "CedentePrestatore" is valid | 00301
```

Reply:
Hai ragione! Il test non è corretto! Il problema si trova nel TerzoIntermediarioOSoggettoEmittente
ad esempio quello che trovi di seguito passa tranquillamente l'SDI, nota IdCodice che gli manca uno zero. Aggiunsto il test per questa casistica che è quella che ho trovato.

        <TerzoIntermediarioOSoggettoEmittente>
            <DatiAnagrafici>
                <IdFiscaleIVA>
                    <IdPaese>IT</IdPaese>
                    <IdCodice>7945211006</IdCodice>
                </IdFiscaleIVA>
                <Anagrafica>
                    <Denominazione>InfoCert S.p.A.</Denominazione>
                </Anagrafica>
            </DatiAnagrafici>
        </TerzoIntermediarioOSoggettoEmittente>